index.js

```diff
@@ -0,0 +1,88 @@
+const { Octokit } = require("@octokit/core");
+const core = require("@actions/core");
+const { Console } = require("console");
+
+function getVersionId(packages, version)  {
+    for (var i=0; i < packages.length; i++) {
+        if (packages[i].name === version) {
+            return packages[i].id;
+        }
+    }
+}
+
+async function FindAndDeletePackageVersion(org, package_type, package_name, version, token) {
```

Comment:
Shouldn't this be `findAndDeletePackageVersion`

Reply:
Will rename it